src/postScores.js

```diff
@@ -0,0 +1,21 @@
+const nameInput = document.getElementById('name');
+const scoreInput = document.getElementById('score');
+
+const postScores = async (url) => {
+  const result = await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      user: nameInput.value,
+      score: scoreInput.value,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+
+  nameInput.value = '';
+  scoreInput.value = '';
+  return result.json();
+};
+
+export default postScores;
```

Comment:
- [ ] Good job so far on defining the `postScore` function and also using the arrow function with the async await pattern :+1: :smile: , But I think there are still some improvements you need to make before moving to the next milestone. 
- I tried adding a new score but I didn't get any result, the page simply refreshes, I think this is due to the fact that you are not handling the form `submit event`. :wave: . Please look into this as part of the project requirement is for users to be able to add new scores from the UI.

- SCREEN RECORD FOR REFERENCE:

https://user-images.githubusercontent.com/107632780/233752316-e55c9263-0086-4b94-a948-897db1bc6f4c.mp4